test/studies/amr/lib/level/LevelSolution_def.chpl

```diff
@@ -19,8 +19,9 @@ class LevelSolution {
   //|\''''''''''''''''''''|\
   //| >    constructor    | >
   //|/....................|/
-  proc LevelSolution(level: Level) {
+  proc init(level: Level) {
     this.level  = level;
+    super.init();
```

Comment:
I'm curious about the placement here versus right after the two field updates following it?  This seems like a good candidate for a Phase 1 default body.